test/evergreen.yml

```diff
@@ -3743,6 +3743,39 @@ buildvariants:
     - name: ".stress-test-4"
     - name: format-abort-recovery-stress-test
 
+- name: ubuntu2004-stress-tests-arm64
+  display_name: Ubuntu 20.04 Stress tests (ARM64)
+  run_on:
+  - ubuntu2004-arm64-large
+  expansions:
+    # Reduce the number of parallel operations, this is to prevent OOM killer interrupting the
+    # format sanitizer stress test.
+    smp_command: -j $(echo "`grep -c ^processor /proc/cpuinfo` - 4" | bc)
+    make_command: ninja
+    test_env_vars:
+      LD_PRELOAD=/usr/lib/aarch64-linux-gnu/libeatmydata.so
+      PATH=/opt/mongodbtoolchain/v3/bin:$PATH
+      WT_TOPDIR=$(git rev-parse --show-toplevel)
+      WT_BUILDDIR=$WT_TOPDIR/cmake_build
+      LD_LIBRARY_PATH=$WT_BUILDDIR:$WT_TOPDIR/TCMALLOC_LIB/lib
+    posix_configure_flags:
+      -DHAVE_DIAGNOSTIC=1
+      -DENABLE_PYTHON=1
+      -DENABLE_ZLIB=1
+      -DENABLE_SNAPPY=1
+      -DENABLE_STRICT=1
+      -DENABLE_STATIC=1
+      -DENABLE_TCMALLOC=1
+      -DCMAKE_PREFIX_PATH="$(pwd)/../TCMALLOC_LIB"
+      -DCMAKE_INSTALL_PREFIX=$(pwd)/LOCAL_INSTALL
+    python_binary: '/opt/mongodbtoolchain/v3/bin/python3'
+  tasks:
+    - name: ".stress-test-1"
+    - name: ".stress-test-2"
+    - name: ".stress-test-3"
+    - name: ".stress-test-4"
+    - name: format-abort-recovery-stress-test
```

Comment:
These would be mapped to a long list of tasks (some of the tasks run as long as 4 hrs). I don't think it's reasonable to have that many tasks running for each commit in the long run (ARM building resources are not as abundant/cost-effective as x86), but probably makes sense for the short term to uncover the issue we are chasing. Can you file a ticket to revisit the scope & frequency of the build variant later on?

Reply:
I added a comment and a WT ticket FIXME to address the issue in the future.